crispy_forms/templatetags/crispy_forms_field.py

```diff
@@ -27,6 +27,11 @@ def is_select(field):
     return isinstance(field.field.widget, forms.Select)
 
 
+@register.filter
+def is_selectsingle(field):
+    return is_select(field) and not field.field.widget.allow_multiple_selected
```

Comment:
Why do we need to limit this to just single selects? The multiple select also supports `custom-select`? From my testing I can't see any difference in the styling on multiple selects between `custom-select` and `form-control`

Reply:
I didn't see any difference either. I thought of minimising the changes in the HTML output (to avoid breaking changes to users). Since Bootstrap documentation says explicitly that it is supported I'm happy to change it an apply let the `custom-select` go to the multiple selects. Actually it might be even less surprising for users. It also have fewer code for us.

It also occurred to me that some other template backend might need this distinction (single vs multiple) but we can cross this bridge when you come to it.

Reply:
https://docs.djangoproject.com/en/3.1/ref/forms/api/#django.forms.BoundField.widget_type Projects using Django 3.1+ can get a similar ability directly in the templates.